spec/plurimath/integration/asciimath_spec.rb

```diff
@@ -262,7 +262,7 @@
 
       it 'returns instance of Asciimath' do
         expect{
-          convert(string_to_formula(string, :asciimath), :asciimath)
+          convert(string_to_formula(string, :latex), :latex)
```

Comment:
@suleman-uzair why is this value now `:latex` when it is expecting an AsciiMath instance?